openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2756,7 +2759,7 @@ The technology described in this specification was made available from contribut
 
    -16
```

Comment:
can you please make sure the version number at the top of the spec is also -16..?

Reply:
I merged current main which includes that change - should be fine now